MvvmCross.Forms/MvvmCross.Forms.csproj

```diff
@@ -24,6 +24,9 @@
     <Compile Include="Platforms\Net\**\*.cs" />
     <Compile Include="Platforms\Wpf\**\*.cs" />
     <Compile Include="Platforms\Console\**\*.cs" />
+    <PackageReference Include="System.Console" Version="4.3.0" />
```

Comment:
Isn't this already in the main library?

Reply:
yeh, I was having some issues just getting it all to build. It should be able to be removed now